test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/PackageLicenseUtilitiesTests.cs

```diff
@@ -148,18 +148,13 @@ public void PackageLicenseUtility_GeneratesLinkForFiles()
             var licenseFileLocation = "License.txt";
             var licenseFileHeader = "header";
             var licenseData = new LicenseMetadata(LicenseType.File, licenseFileLocation, null, null, LicenseMetadata.CurrentVersion);
-            var licenseContent = "License content";
+            //TODO:  var licenseContent = "License content";
```

Comment:
Can you add the url to a tracking issue please

Reply:
will fix as part of this PR. thx for noticing.